test/e2e/nodeportlocal_test.go

```diff
@@ -476,8 +477,8 @@ func NPLTestPodAddMultiProtocol(t *testing.T, data *TestData) {
 	clientNode := nodeName(0)
 	serverNode := nodeName(1)
 	if len(clusterInfo.windowsNodes) != 0 {
-		serverNode = workerNodeName(clusterInfo.windowsNodes[0])
```

Comment:
Do you think we could use a general function to find serverNode and clientNode? The there is no need to change the similar logic in every function?

Reply:
Added a new function to find clientNode and serverNode